pkg/common/tpm_attestor_test.go

```diff
@@ -0,0 +1,84 @@
+package common
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestEncodeDecode(t *testing.T) {
+	testCases := []struct {
+		name     string
+		hasCert  bool
+		pemBytes []byte
+		pubHash  string
+	}{
+		{
+			name:    "certificate",
+			hasCert: true,
+			pemBytes: []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
Could you add some notes on how to generate this test data? Can be top-level docs in the README or comments above the places where test data exists.

Reply:
:+1: added openssl commands as comments